dev/TEAL_opcodes.md

```diff
@@ -1304,11 +1318,11 @@ bitlen interprets arrays as big-endian integers, unlike setbit/getbit
 - Opcode: 0xb1
 - Pops: _None_
 - Pushes: _None_
-- begin preparation of a new inner transaction
+- begin preparation of a new inner transaction in a new transaction group
 - LogicSigVersion >= 5
 - Mode: Application
 
-`itxn_begin` initializes Sender to the application address; Fee to the minimum allowable, taking into account MinTxnFee and credit from overpaying in earlier transactions; FirstValid/LastValid to the values in the top-level transaction, and all other fields to zero values.
+`itxn_begin` initializes Sender to the application address; Fee to the minimum allowable, taking into account MinTxnFee and credit from overpaying in earlier transactions; FirstValid/LastValid to the values in the invoking transaction, and all other fields to zero values.
```

Comment:
Just checking: the `txn.Accounts`, `txn.ForeignAssets`, `txn.ForeignApplications` are set to empty, not the arrays in the invoking txn, right?

Reply:
Correct. They begin empty, after `itxn_begin` or `itxn_next`, but can be added to with `itxn_field`.  Those additions must be of resources that are, themselves, accessible by the calling transaction.